airflow/utils/state.py

```diff
@@ -105,6 +107,7 @@ class State:
         TaskInstanceState.RUNNING: 'lime',
         TaskInstanceState.SUCCESS: 'green',
         TaskInstanceState.SHUTDOWN: 'blue',
+        TaskInstanceState.RESTARTING: 'goldenrod',
```

Comment:
Can we do a different color? `goldenrod` is too similar `gold`, `orange` and `tan`

We don't use many blues, violets, or browns. [Here](https://www.w3schools.com/cssref/css_colors.asp) are browser supported colors to choose from.

Reply:
Thanks. Changed to `violet`.